core/src/main/scala/com/microsoft/azure/synapse/ml/param/TypedArrayParam.scala

```diff
@@ -16,26 +16,49 @@ class TypedArrayParam[T: TypeTag](parent: Params,
                          doc: String,
                          isValid: Seq[T] => Boolean = (_: Seq[T]) => true)
                         (@transient implicit val dataFormat: JsonFormat[T])
-  extends JsonEncodableParam[Seq[T]](parent, name, doc, isValid) with DotnetWrappableParam[Seq[T]] {
+  extends JsonEncodableParam[Seq[T]](parent, name, doc, isValid)
+    with DotnetWrappableParam[Seq[T]] with RWrappableParam[Seq[T]] {
   type ValueType = T
 
   def w(v: java.util.ArrayList[T]): ParamPair[Seq[T]] = w(v.asScala)
 
   // TODO: Implement render for this
   override private[ml] def dotnetTestValue(v: Seq[T]): String = {
-    throw new NotImplementedError(s"No translation found for this TypedArrayParame: $v")
+    throw new NotImplementedError(s"No translation found for this TypedArrayParam: $v")
   }
 
   override private[ml] def dotnetTestSetterLine(v: Seq[T]): String = {
     typeOf[T].toString match {
-      case t if t == "Seq[com.microsoft.azure.synapse.ml.explainers.ICECategoricalFeature]" =>
```

Comment:
@serena-ruan I found that the Seq[] part was not actually present, so I updated this. Also, if you see the R counterpart below, it seems like just printing the elements of the param should work, so that you don't need to throw in dotnetTestValue

Reply:
BTW: thanks for pointing me to this. I didn't see how I could use it at first, but in the end it worked out really well.